core/src/it/scala/cr/pulsar/PulsarSpec.scala

```diff
@@ -64,7 +64,7 @@ class PulsarSpec extends PulsarSuite {
               val produce =
                 Stream(testEvent)
                   .covary[IO]
-                  .evalMap(producer.send)
+                  .evalMap(producer.send(_))
```

Comment:
I know this is just aesthetics but this looks ugly to me :smile:

Reply:
Oh it's because `send` has now two arguments...

Reply:
Yeah, that's why I was thinking about adding a separate method, it's even worse with `send_(_)`